reporting_app/templates/proc_report.html

```diff
@@ -56,24 +36,9 @@
 
 
 {% macro proc_report(procs) -%}
-{# procs (list[dict]) - analysis_driver_procs from the Rest API. #}
-
-<h2><a data-toggle="collapse" data-target="#analysis_driver_proc_tabs">Analysis Driver processes</a></h2>
-<div id="analysis_driver_proc_tabs" class="tab-content collapse">
-    <p>Number of times processed: {{ procs|length }}</p>
-    {% if procs %}
-        <ul class="nav nav-tabs" id="analysis_driver_proc_ul">
-            <li class="nav active"><a data-target="#analysis_driver_proc_1_tab" data-toggle="tab">{{ procs[0]._created }}</a></li>
-            {% for proc in procs[1:] %}
-                <li class="nav"><a data-target="#analysis_driver_proc_{{ loop.index + 1 }}_tab" data-toggle="tab">{{ proc._created }}</a></li>
-            {% endfor %}
-        </ul>
-
-        <div id="analysis_driver_proc_1_tab" class="tab-pane fade in active">{{ proc_timeline(procs[0]) }}</div>
-        {% for proc in procs[1:] %}
-            <div id="analysis_driver_proc_{{ loop.index + 1 }}_tab" class="tab-pane fade">{{ proc_timeline(proc) }}</div>
-        {% endfor %}
-    {% endif %}
-</div>
-
+<br/>
+<p>Number of times processed: {{ procs|length }}</p>
+{% call(tab) tabset('analysis_driver_procs', 'Analysis Driver processes', procs, '_created') %}
```

Comment:
The new marco is showing the analysis_driver_procs section by default where it used to be hidden.
Adding an option to that effect in the macro should fix this
```python
tabset('analysis_driver_procs', 'Analysis Driver processes', procs, '_created', default_collapsed='')
...
{% macro tabset(name, title, tabs, title_field='title', default_collapsed='in') -%}
...
<div id="{{ name }}_tabs" class="tab-content collapse {{ default_collapsed }}">

```